runtime/testy/ansible_pb2.py

```diff
@@ -0,0 +1,166 @@
+# Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
:cry: In that case, can you report a github issue, and make a note that compiled protobuf files will need to be removed when the issue is fixed?